src/main/java/astor/Storage.java

```diff
@@ -0,0 +1,137 @@
+package astor;
+
+import astor.task.Deadline;
+import astor.task.Event;
+import astor.task.Task;
+import astor.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Represents a storage manager.
+ *
+ * The storage manager connects to datafile and creates it if absent,
+ * writes to datafile, reads from datafile.
+ */
+public class Storage {
+
+    private FileWriter fw;
```

Comment:
Perhaps this could be more explicitly spelt out? fw may not be clear later in the code